package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Front End Capstone for Hack Reactor",
   "main": "index.js",
   "scripts": {
+    "start": "nodemon server/index.js",
```

Comment:
this is duplicative of the server-dev script. Can we pick one and ensure the readme is updated to whichever one we choose?